### ISSUE TITLE
Tab should remain the same when returning to a Finto-page from external site on the same tab (Metatietosanasto)

### ISSUE THREAD

**Issue by fantsby-m** [NatLibFi/Skosmos#1595]
When you click an external link on a concept page in Finto and it opens on the same browser tab (as they do in Finto) and then return to the same Finto-page using backspace, you always land on the Hierarkia-tab on Finto instead of the Finto-tab on which you clicked the external link. This is a usability issue especially on Metatietosanasto since it does not have an ontology-type hierarchy, i.e. the Hierarkia-tab is not in use - even though it is visible in Metatietosanasto as well. This may be an issue in other vocabularies and ontologies as well.

For example, if you click the RDA Vocabularies link on this page
![kuva](https://github.com/NatLibFi/Skosmos/assets/77340458/e13fc35e-34f3-487f-9d7d-1e817f025d1e)

and then return using backspace, you land on this:
![kuva](https://github.com/NatLibFi/Skosmos/assets/77340458/1e0265b9-d3a9-45da-a190-07cd78f14579)

Solutions:
1) the tab when you return remains the tab you left from, or
2) external links open in new browser tab instead of the same browser tab, leaving the Finto page from which you click the link intact.

**Comment by osma** [comment]
I cannot reproduce this directly. However, a slightly different sequence of actions gives the same result:

1. Navigate to a regular concept (not group) using the Groups tab in MTS
2. Refresh the page

The refresh causes a switch to the Hierarchy tab, which is surprising from the user perspective.

**Comment by fantsby-m** [comment]
Yes, the sequence in the images is the same, first navigate to a concept through the group tab, then click the RDA Vocabularies link, then hit backspace - you end to the hierarchy tab.

**Comment by joelit** [comment]
We'll take this case into accout when designing the group tab in Skosmos 3, but it's unlikely we can  take time to release a fix in Skosmos 2, sadly.
